build.gradle.kts

```diff
@@ -52,6 +52,12 @@ val compileJavaJvmArgs by extra((project.properties["compileJavaJvmArgs"] as Str
 val shouldRunAllCheckers by extra(isCI || project.hasProperty("runAllCheckers"))
 
 tasks.register<UpdateEapBuildNumber>("updateEapBuildNumber")
+tasks.register<UpdateIntellijStableVersions>("updateIntellijStableVersions")
+
+tasks.register("updateIntellijVersions") {
+  dependsOn("updateIntellijStableVersions", "updateEapBuildNumber")
```

Comment:
TBD but I think that it doesn't even make much sense to keep the two tasks around, I doubt we'll ever need to run just one of them (it might in fact lead to unexpected results sometimes, esp. when e.g. we keep updating EAP for 223 once 2022.3 is released)